client/state/happychat/selectors/is-presales-zendesk-chat-available.js

```diff
@@ -0,0 +1,13 @@
+import { get } from 'lodash';
+
+import 'calypso/state/happychat/init';
+
+/**
+ * Returns if presales zendesk chat is available.
+ *
+ * @param   {object}  state  Global state tree
+ * @returns {boolean}        true, when presales_zendesk is available
+ */
+export default function isPresalesZendeskChatAvailable( state ) {
+	return get( state, 'happychat.user.availability.presale_zendesk', false );
```

Comment:
There's a project by the calypso team to remove lodash from calypso for various reasons. Could this use native JS instead?

```suggestion
	return happychat?.user?.availability?.presale_zendesk ?? false;
```